src/transformStateWithClones.js

```diff
@@ -8,6 +8,34 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const ADD_PROPERTIES = 'addProperties';
+  const REMOVE_PROPERTIES = 'removeProperties';
+  const CLEAR = 'clear';
+  const newState = { ...state };
+  const arrState = [];
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
Use a destruction for `action`
```suggestion
  for (const action of actions) {
    const { type, extraData, keysToRemove } = action;
  
    switch (type) {
```